packages/flutter/lib/src/services/text_input.dart

```diff
@@ -484,11 +495,24 @@ class TextInputConfiguration {
   /// Defaults to false.
   final bool readOnly;
 
+  @Deprecated(
+    'use obscure instead.'
```

Comment:
Here and below, capitalize the "u", and change "obscure" to "obscureTextBehavior".